packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -38,6 +38,21 @@ const metadata: Meta<SelectArgs> = {
     },
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
+        <!--
```

Comment:
I played with the Storybook in Firefox, Edge, and Safari on mac. The combo box and select work great in all of them!

But now the menu button is the odd one out, still clipping its contents inside its container on the Docs page. I assume that should work since it uses anchored region already? Could you apply the same workaround in its story?

Reply:
And since that will mean this style is duplicated several times, you could start sharing it by following the pattern we've started in `packages/nimble-components/src/utilities/tests/storybook.ts` for `usageWarning` and `overrideWarning`.

Reply:
Done